### ISSUE TITLE
How to create a concrete function for getting coordinates of a masked image and convert it to tensorflow lite model

### ISSUE THREAD

**Issue by chandramtv** [tensorflow/tensorflow#31595]
How to create concrete function in tensorflow to get semantic segmentation image and calculate co-ordinates.

We are having current code for the same but not able to concrete function as we wanted to convert in tflite file for mobile application.

Existing normal code is given below:
def predict_new_image(img_path, model):
img = load_img(img_path, grayscale=True)
x_img = img_to_array(img)
x_img = resize(x_img, (128, 128, 1), mode='constant', preserve_range=True)

X = np.zeros((1, 128, 128, 1), dtype=np.float32)
y_img = np.zeros((1, 128, 128, 1), dtype=np.float32)
X[0, ..., 0] = x_img.squeeze() / 255

pred = model.predict(X)
preds_img = (pred > 0.5).astype(np.uint8)

img_arr = preds_img[:,:,0]

# get the coordinates where the pixel isn't white (at a threshold)
black_thres = 1
idx = [(i,j) for i,x in enumerate(img_arr) for j,y in enumerate(x) if img_arr[i,j]==black_thres]
return idx

**Comment by ravikyram** [comment]
Please provide details about what platform you are using (operating system, architecture). Also include your TensorFlow version. Also, did you compile from source or install a binary?

Make sure you also include the exact command if possible to produce the output included in your test case. If you are unclear what to include see the issue template displayed in [the Github new issue template](https://github.com/tensorflow/tensorflow/issues/new/choose).

We ask for this in the issue submission template, because it is really difficult to help without that information. Thanks!

**Comment by puneethmishra** [comment]
We are using a UNET architecture for medical image segmentation. Using the keras version present within tensorflow 2.0. trying to compile from the source . The model gets packaged well within the TFLite. But not able to package the additional code for calculating coordinates of the masked image.

**Comment by ravikyram** [comment]
Please, go through below link and see if it helps you to create a concrete function and convert to tensorflowlite. Thanks!
https://github.com/tensorflow/tensorflow/issues/26708

**Comment by ravikyram** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
